modules/auxiliary/scanner/dcerpc/petitpotam.rb

```diff
@@ -9,6 +9,12 @@
 require 'ruby_smb/dcerpc/lsarpc'
 require 'ruby_smb/dcerpc/efsrpc'
 
+module RubySMB::Dcerpc::EfsrpcOverLsarpc
```

Comment:
If we want this to be accessible from RubySMB then we should probably move it there. I think this is more of an edge case though for petitpotam though so it make make more sense to just define an `EfsrpcOverLsarpc` module in the `MetasploitModule` class to avoid altering the global namespace.

Reply:
Yes, i makes sense. I've addressed this in [b3c2ae4](https://github.com/rapid7/metasploit-framework/pull/19833/commits/b3c2ae4f5195232963a9b32f013d5f335344a446). Thank you!